test/classes/initializers/where-clause4.good

```diff
@@ -0,0 +1,2 @@
+where-clause4.chpl:12: error: unresolved call '_new(type Foo, 3.4)'
```

Comment:
It's tempting to say that there should also be a "future" version of this test to reflect the
notion that this error message isn't all that great.